pom.xml

```diff
@@ -228,7 +228,7 @@
         <rhinoVersion>1.7R4</rhinoVersion>
         <rhinoJsr223Version>1.1</rhinoJsr223Version>
         <saajVersion>1.3</saajVersion>
-        <saxonVersion>9.6.0-1</saxonVersion>
+        <saxonVersion>9.6.0-7</saxonVersion>
```

Comment:
This PR will fix a few issues reported and solved in Saxon 9.6, for Saxon 9.7 and XPath 3.1 I have created another enhancement MULE-9452.